frontend/pages/index.html

```diff
@@ -1,11 +1,50 @@
+
 <!DOCTYPE html>
```

Comment:
Could this be in a seperate file? e.g create a new file `leaderboard.html`, along with a `leaderboard.css`. And then keep this file as it was (you can just copy paste the original content from the main branch of this repo)